framework/test/unit/modules/interoperability/mainchain/store.spec.ts

```diff
@@ -102,41 +108,91 @@ describe('Mainchain interoperability store', () => {
 			crossChainCommand: CROSS_CHAIN_COMMAND_NAME_REGISTRATION,
 			sendingChainID: utils.intToBuffer(2, 4),
 			receivingChainID: utils.intToBuffer(3, 4),
-			fee: BigInt(1),
-			status: 1,
+			fee: BigInt(100000),
+			status: CCM_STATUS_OK,
 			params: Buffer.alloc(0),
 		};
 
 		const newCCM = {
 			...ccm,
 			sendingChainID: ccm.receivingChainID,
 			receivingChainID: ccm.sendingChainID,
-			status: CCM_STATUS_CHANNEL_UNAVAILABLE,
 		};
 
+		const ccmBounceContext = {
+			ccm,
+			newCCMStatus: CCM_STATUS_OK,
+			ccmProcessedEventCode: 0,
+			eventQueue: new EventQueue(0),
+		};
+
+		const ccmID = utils.hash(codec.encode(ccmSchema, ccm));
+		const minimumFee = MIN_RETURN_FEE * BigInt(ccmID.length);
+		let ccmProcessedEvent: CcmProcessedEvent;
+		let ccmSendSuccessEvent: CcmSendSuccessEvent;
+
 		beforeEach(() => {
+			ccmProcessedEvent = mainchainInteroperabilityStore.events.get(CcmProcessedEvent);
+			ccmSendSuccessEvent = mainchainInteroperabilityStore.events.get(CcmSendSuccessEvent);
+			jest.spyOn(ccmProcessedEvent, 'log');
+			jest.spyOn(ccmSendSuccessEvent, 'log');
 			mainchainInteroperabilityStore.addToOutbox = jest.fn();
 		});
 
-		it('should not call addToOutbox if terminatedStateAccount exists', async () => {
-			// Arrange
-			await terminatedStateSubstore.set(context, ccm.sendingChainID, chainAccount);
-
+		it(`should not call addToOutbox if ccm status is not equal to ${CCM_STATUS_OK}`, async () => {
```

Comment:
I think we are missing assertion for the event CCM_PROCESSED_RESULT_DISCARDED emitted